default/stringtables/ru.txt

```diff
@@ -5860,11 +5860,17 @@ SITREP_SHOWALL_TEMPLATES
 SITREP_WELCOME_LABEL
 Добро пожаловать
 
+SITREP_SYSTEM_GOT_INCOMING_WARNING
+%system%: Вражеские корабли прибывают на следующем ходу!
+
+SITREP_SYSTEM_GOT_INCOMING_WARNING_LABEL
+Прибывающие враги
+
 SITREP_GAS_GIANT_GENERATION_REMINDER
-%planet% является идеальным местом для %buildingtype%, но никто еще ничего там не строил
+%planet% является идеальным местом для %buildingtype%, но там ещё ничего не построено.
```

Comment:
Is adding periods at the end of everything stylistically better in russian?

Reply:
Trailing punctuation (period, exclamation mark etc.) is word standard for sentences. 

Should it be added to each translation entry or one time to the siterep template, is an open question.

Reply:
Just wanted to correspond to english original + my perfectionism :D It's no more matter as well as in other languages, like English.